it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -355,17 +356,18 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
             )
           )
           stubStoreIdentifiersMatch(testJourneyId, identifiersMatch = false)(status = OK)
-          stubStoreBusinessVerificationStatus(testJourneyId, BusinessVerificationUnchallenged)(status = OK)
+          stubStoreBusinessVerificationStatus(testJourneyId, BusinessVerificationNotEnoughInformationToCallBV)(status = OK)
           stubStoreRegistrationStatus(testJourneyId, RegistrationNotCalled)(status = OK)
-          stubRetrieveBusinessVerificationStatus(testJourneyId)(status = OK, body = testBusinessVerificationUnchallengedJson)
+
+          stubRetrieveBusinessVerificationStatus(testJourneyId)(status = OK, body = testBusinessVerificationJson(value = businessVerificationNotEnoughInfoToChallengeKey))
```

Comment:
Should we be using BusinessVerificationNotEnoughInformationToCallBVKey here?

Reply:
you are right